### PR TITLE
OSSM-3353 Implement installation of 3.0 charts

### PR DESCRIPTION
**Pull request by dgn**

### CODE REVIEW

File: deploy/maistra-operator.yaml [hunk 1]
```diff
@@ -10728,51 +10728,15 @@ spec:
...
-        olm.relatedImage.v2_3.proxyv2: quay.io/maistra/proxyv2-ubi8:2.3.0
-        olm.relatedImage.v2_3.rls: quay.io/maistra/ratelimit-ubi8:2.3.0
-
-        olm.relatedImage.v2_4.cni: quay.io/maistra/istio-cni-ubi8:2.4.0
-        olm.relatedImage.v2_4.grafana: quay.io/maistra/grafana-ubi8:2.4.0
-        olm.relatedImage.v2_4.pilot: quay.io/maistra/pilot-ubi8:2.4.0
-        olm.relatedImage.v2_4.prometheus: quay.io/maistra/prometheus-ubi8:2.4.0
-        olm.relatedImage.v2_4.proxyv2: quay.io/maistra/proxyv2-ubi8:2.4.0
-        olm.relatedImage.v2_4.rls: quay.io/maistra/ratelimit-ubi8:2.4.0
+        olm.relatedImage.v3_0.cni: quay.io/maistra/istio-cni-ubi8:3.0.0
```
Comment (jwendell) [inline deploy/maistra-operator.yaml]: ubi9?

File: deploy/maistra-operator.yaml [hunk 2]
```diff
@@ -10728,51 +10728,15 @@ spec:
...
+        olm.relatedImage.v3_0.pilot: quay.io/maistra/pilot-ubi8:3.0.0
+        olm.relatedImage.v3_0.proxyv2: quay.io/maistra/proxyv2-ubi8:3.0.0
 
     spec:
       serviceAccountName: istio-operator
       containers:
       - name: istio-operator
-        image: quay.io/maistra/istio-ubi8-operator:2.4.0
+        image: quay.io/maistra/istio-ubi8-operator:3.0.0
```
Comment (jwendell) [inline deploy/maistra-operator.yaml]: ubi9?

File: deploy/servicemesh-operator.yaml [hunk 3]
```diff
@@ -10728,52 +10728,10 @@ spec:
...
-        olm.relatedImage.v2_4.pilot: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-pilot-rhel8:${OSSM_24}
-        olm.relatedImage.v2_4.prometheus: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-prometheus-rhel8:${OSSM_24}
-        olm.relatedImage.v2_4.proxyv2: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-proxyv2-rhel8:${OSSM_24}
-        olm.relatedImage.v2_4.rls: registry-proxy.engineering.redhat.com/rh-osbs/openshift-service-mesh-ratelimit-rhel8:${OSSM_24}
-        # 3.0 images
         olm.relatedImage.v3_0.cni: ${OSSM_CNI_IMAGE}
         olm.relatedImage.v3_0.pilot: ${OSSM_PILOT_IMAGE}
-        olm.relatedImage.v3_0.proxyv2: ${OSSM_PROXY_IMAGE}
-        olm.relatedImage.v3_0.rls: ${OSSM_RATELIMIT_IMAGE}
+        # FIXME: replace with our own build once we have one
```
Comment (jwendell) [inline deploy/servicemesh-operator.yaml]: nit: create a blocker issue in jira so we don't forget about it :)

Comment (dgn) [reply deploy/servicemesh-operator.yaml]: done! https://issues.redhat.com/browse/OSSM-3624

File: deploy/maistra-operator.yaml (hunk 1 above)
Comment (dgn) [reply deploy/maistra-operator.yaml]: done

**Review by jwendell — APPROVED** [review]

**Pull request merged**